data.json

```
@@ -1,4 +1,48 @@
 [
+  {
+    "command": "",
+    "description": "A package built to be a fast and flexible pure-Julia library for handling delimited text files.",
+    "kind": "header",
+    "package": "CSV",
+    "tags": [
+      "csv",
+      "delimited-files"
+    ]
+  },
+  {
+    "command": "CSV.Rows(source)",
+    "description": "Read a csv input and return a CSV.Rows object.",
+    "kind": "snippet",
+    "package": "CSV",
+    "tags": [
+      "csv",
+      "delimited-files",
+      "rows"
+    ]
+  },
+  {
+    "command": "CSV.File(source)",
+    "description": "Read a UTF-8 CSV input and return a CSV.File object.",
```

```suggestion
    "description": "Read a CSV input.",
```
You used CSV here but csv before. Either all CAPS or no caps. Also, I simplified the description to make it beginner-friendly.